src/spock.c

```diff
@@ -1108,11 +1108,11 @@ _PG_init(void)
 							"This setting is deprecated and has no effect. "
 							"The replay queue now dynamically allocates memory as needed.",
 							&spock_replay_queue_size,
-							4194304,
+							4,
 							0,
-							INT_MAX,
+							MAX_KILOBYTES / 1024,
```

Comment:
What is reason to change that?

Reply:
It is just because of boring complain in the log. We assign this GUC in terms of megabytes, so we need to declare units and proper limits too.